operation/fhir-operation-member-match/src/main/java/com/ibm/fhir/operation/davinci/hrex/provider/MemberMatchFactory.java

```diff
@@ -48,12 +51,13 @@ public static MemberMatchFactory factory() {
      * Gets the strategy for this specific configuration.
      * @param config
      * @return the MemberMatchStrategy or the DefaultMemberMatchStrategy
+     * @throws FHIROperationException
      */
-    public MemberMatchStrategy getStrategy(ConfigurationAdapter config) {
+    public MemberMatchStrategy getStrategy(ConfigurationAdapter config) throws FHIROperationException {
         String key = config.getStrategyKey();
         MemberMatchStrategy strategy = keyToMemberMatchStrategy.get(key);
         if (strategy == null) {
-            strategy = keyToMemberMatchStrategy.get(DEFAULT_STRATEGY);
```

Comment:
the default strategy is still the default though, right?  so this would only apply if someone has explicitly configured a strategy that is not in the config?

Reply:
Correct